mne_bids/read.py

```diff
@@ -652,45 +653,66 @@ def get_matched_empty_room(bids_basename, bids_root):
         The basename corresponding to the best-matching empty-room measurement.
         Returns None if none was found.
     """
-    kind = 'meg'
+    from mne_bids import make_bids_basename  # Avoid circular import.
```

Comment:
shouldn't we move this to a `utils.py` file? It keeps coming up in different PRs

Reply:
Made a PR: #424 
Will rebase this branch once #424 has been merged.